fud/fud/stages/interpreter.py

```diff
@@ -195,3 +212,93 @@ def convert(x):
 
     with out_path.open("w") as f:
         sjson.dump(output_json, f, indent=2, use_decimal=True)
+
+
+def parse_from_json(output_data_str, original_data_file_path):
```

Comment:
Correct me if I’m wrong but there seems to be some duplication in this code and the code in json to dat? Can we combine them and have all the data conversion be in the same place so that future changes to the data format are easier?

Reply:
There's probably some overlap. It would be good to overhaul all this code into a unified data format surrounding fud, but I think that should probably have its own issue / PR.